tests/post/steps/test_volume.py

```diff
@@ -206,6 +206,28 @@ def storage_class_exist(name, sc_client):
         sc_client.create_from_yaml(kube_utils.DEFAULT_SC.format(name=name))
 
 
+@given(
+    parsers.parse(
+        "a LVM LV '{name}' in VG '{vg_name}' was created and formatted, then removed"
+    )
+)
+def lvm_lv_was_used(host, name, vg_name):
+    with host.sudo():
+        host.run_test(f"vgdisplay {vg_name}")
+        host.run_test(f"lvcreate -L 1G -n {name} {vg_name}")
+        host.run_test(f"mkfs.ext4 /dev/{vg_name}/{name}")
+        host.run_test(f"lvremove -y /dev/{vg_name}/{name}")
+
+    yield
+
+    with host.sudo():
+        lv_exists = host.run(f"lvdisplay /dev/{vg_name}/{name}")
+        if lv_exists.rc != 0:
+            # LV does not exist, attempt to recreate for wiping the signature
+            host.run_test(f"lvcreate -L 1G -n {name} -y {vg_name}")
```

Comment:
The wipefs is missing here I think

Reply:
Oh yes, I need an `else` for this (because the `lvcreate -y` will handle the wipe)